_integration-schemas/adroll/v1/campaigns.md

```diff
@@ -0,0 +1,142 @@
+---
+tap: "adroll"
+version: "1"
+key: ""
+
+name: "campaigns"
+doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-campaign-get"
+singer-schema: "https://github.com/singer-io/tap-adroll/blob/master/tap_adroll/schemas/campaigns.json"
+description: |
+  The `{{ table.name }}` table contains information about the advertising campaigns in your {{ integration.display_name }} account.
+
+replication-method: "Full Table"
+
+api-method:
+    name: "getCampaigns"
+    doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-campaign-get"
```

Comment:
Similar issue to the endpoint in `ad_groups`, this isn't the endpoint the tap uses to get this data.